infrastructure/cdn-in-a-box/Makefile

```diff
@@ -73,6 +75,8 @@ traffic_router/tomcat.rpm: ../../dist/tomcat-$(SPECIAL_SEASONING)
 	cp -f $? $@
 traffic_stats/traffic_stats.rpm: ../../dist/traffic_stats-$(SPECIAL_SAUCE)
 	cp -f $? $@
+../../traffic_ops/ort/atstccfg/atstccfg: $(ORT_SOURCE)
+	go build -o $@ ../../traffic_ops/ort/atstccfg
```

Comment:
The ORT RPM not only includes `traffic_ops_ort.pl` but also requires systemd support. Which is why we weren't doing it that way in the first place.

Reply:
Yes `traffic_ops_ort.pl` requires systemd support, but the ORT RPM itself does not. `traffic_ops_ort.pl` itself is 91KB (less when compressed inside the RPM), and considering it (and `supermicro_udev_mapper.pl`) can be removed from the image in the same layer that the RPM is installed, the existence of `traffic_ops_ort.pl` in the RPM is not alone enough of a reason to build from source in the Makefile.

Installing the RPM with no dependencies seems reasonable to me, we don't need Perl anything installed in order to run ATSTCCFG.

Reply:
I was really hoping to get away with not installing the RPM here, because I think that taking the time to build an entire RPM just to unwrap it for one binary is a waste of time for a dev environment, and this is the only image where it hasn't been done in the past (besides the non-ATC ones).

Reply:
Alternatively, installing the ORT RPM helps lay the groundwork for actually running the official ORT-thing in CIAB. Since ORT is will be undergoing a redesign/rewrite in the near future, it might be best to start planning on getting it into CIAB.

Reply:
I get what you are saying about the time taken to build the ORT RPM though, since it is build alongside the Traffic Ops RPM. #2776 (separate ORT build) seems like a good way to speed it up. But because these PRs are pretty much independent, #2776 shouldn't hold up this PR.

Reply:
> _"Alternatively, installing the ORT RPM helps lay the groundwork for actually running the official ORT-thing in CIAB. Since ORT is will be undergoing a redesign/rewrite in the near future, it might be best to start planning on getting it into CIAB."_

Only if we're going to install _that_ by RPM. If it were up to me, there would be no RPMs and things would build directly from the source tree. I know that's not popular because of how CIAB has to flex between being a testing and a development environment, but that's just how I feel.

Reply:
I believe the ORT replacement will be installed via RPM -- the 5, 10, 20, however many binaries/things there are going to be, should all be included in the same RPM. At least, that's what I understand out of the blueprint review so far.

Reply:
No, I mean there will _be_ an RPM, but the RPM just contains the built binaries. So what I'm saying is just build the binaries and pull those into CiaB instead of building the binaries, putting them in an RPM, pulling that into CiaB, then unwrapping the RPM and pulling out the binaries.

Reply:
Building the binary directly like this is ignoring the entire build environment that is set up via the docker build -- things like the particular version of Go used to build the binary, etc. This assumes you have `go` installed locally, which is not always the case, not to mention the version you have installed locally might not even match the "official" version used in the rpm build container. Also, if the CPU architecture from where you ran `make` differs from the CPU architecture of the container it's used in, it's not going to work in the container.

Reply:
None of that's a problem unless CiaB is being used for testing deployments. As a dev environment it works fine.

Reply:
As a dev environment it will only work fine if your CPU architecture matches the container's CPU architecture, right?

Reply:
The container's CPU architecture can only match your CPU architecture. Idk how you would break that.

But actually, I didn't make this clear: CiaB today _is_ used for testing and so whatever I'd like for the future, having the right Go version etc. _is_ important today.

I was waiting for the "ORT has its own build" PR to merge before updating this so it can use that new build command to generate the RPM. However, I just thought of something: what if I just build it from source from within the `cache/Dockerfile` container? Still no unnecessary RPM work, but we get the correct environment. Would everyone be good with that.

Reply:
> The container's CPU architecture can only match your CPU architecture. Idk how you would break that.

I assume that the VM used by Docker For Mac might have a different virtualized CPU architecture than the underlying Mac, but I could be wrong about that and don't feel like verifying. Either way, something built on Mac cannot be assumed to run correctly on Linux.

> Would everyone be good with that? 

That's better but still not ideal. I don't really see why using the RPM is a problem. The RPM would only need rebuilt if you were testing changes to atstccfg, but it doesn't really make sense to develop atstccfg in CIAB when CIAB isn't using traffic_ops_ort.pl.

Reply:
Yeah I guess that's true.

I still don't like using RPMs, but I feel more strongly about just getting atstccfg to run inside CiaB. So I'm just gonna put this into a draft status pending the merge of #2776 .

Reply:
Alright, it's done.